store/sagas.js

```diff
@@ -1,20 +1,22 @@
-import { all, fork } from 'redux-saga/effects';
+import { all } from 'redux-saga/effects';
 
 import networkConnectivitySaga from './sagas/networkConnectivity.sagas';
 import requestSaga from './sagas/request.sagas';
 import userSaga from './sagas/user.sagas';
+/* eslint-disable */
+import contactsSaga from './sagas/contacts.sagas';
+/* eslint-enable */
```

Comment:
Some time ago, when I was committing, the eslint detected an error with this file. I added the annotations because the error was not really clear. At this time I delete the annotations, execute the commit and the file passed the eslint verification.